packages/xstate-test/src/TestModel.ts

```diff
@@ -102,7 +108,9 @@ export class TestModel<TState, TEvent extends EventObject> {
     let minWeight = Infinity;
     let shortestPaths: Array<TestPath<TState, TEvent>> = [];
 
-    const paths = this.filterPathsTo(statePredicate, this.getShortestPaths());
+    const paths = deduplicatePaths(
+      this.filterPathsTo(statePredicate, this._getStatePaths())
```

Comment:
`this._getStatePaths` piggybacks on `TestModel.defaults.pathGenerator` so this is still using shortest paths - but perhaps I should make this more explicit